### DISCUSSION TITLE
How do I control the order of stashed windows?

### DISCUSSION THREAD

**Discussion by helicos6** [rxhanson/RectanglePro-Community#517] [Q&A]
I have 3 windows that I usually stash to the right side of the screen.

I want each window to be stashed in the same area every time. However, what happens is 2 of the apps get randomly stashed in the top and the middle positions. So I'm always guessing (and guessing wrong often) the right window to unstash.

How can I make sure the same window gets stashed in the top most region of the right side screen border, for example?

**Comment by rxhanson** [comment]
The way it currently works, the only solution is to enable color tabs. I'm open to ideas for improvement here, if there's a reason you've tried color tabs and don't like them.

**Comment by helicos6** [comment]
I do have color tabs enabled and I like them. My question is:

If I have 2 windows (2 different apps) of the same height (both windows go from the menu bar to the bottom of the screen), how do I make sure the top color tab on the right side of the screen is always the same specific app window that is stashed? And the middle color tab is also always the same other app window?

The bottom color tab on the right side of the screen is never an issue because this app's height is from the middle of the screen to the bottom so it's always the same app that gets stashed there.

**Reply by rxhanson** [reply]
Got it, thanks for the follow up. Let me revisit this logic and I'll see what I can do.
